cli/plugins/test/index.js

```diff
@@ -1,61 +1,86 @@
 'use strict';
 const pa11y = require('pa11y');
+const { Analyzer } = require('hint');
 
 /**
  * Test implements Joy commands for running a11y and static code tests
- * 
+ *
  */
 class Test {
-
   /**
    * @param {*} joy : Instance of a Joy class c/w config.json, env and helper functions
-   * 
+   *
    */
   constructor(joy) {
     const prog = joy.prog;
     prog
       .command('test', 'Run various tests including a11y and static analysis')
-      .argument('<subcommand>', 'subcommand [a11y, static]', ['a11y', 'static'])
+      .argument('<subcommand>', 'subcommand [a11y, staticServer, staticWeb]', ['a11y', 'staticServer', 'staticWeb'])
```

Comment:
What are your thoughts on `camelCase` vs `lower-dash-separated` for the arguments?  Any pros/cons?

Reply:
I just discovered that Caporal will automatically convert dash-separated to dashSeparated when it passes the value in to code. So, I will start moving toward dash-separated as I touch each interface.